README.md

```diff
@@ -27,28 +22,52 @@ discord.js is a powerful [Node.js](https://nodejs.org) module that allows you to
 - 100% coverage of the Discord API
 
 ## Installation
+
 **Node.js 8.0.0 or newer is required.**  
 Ignore any warnings about unmet peer dependencies, as they're all optional.
 
-Without voice support: `npm install discord.js`  
```

Comment:
I don't like the changes to this section, as it looks a lot less clean. People who are using yarn know how to install packages.